cmd/kubeturbo/app/kubeturbo_builder.go

```diff
@@ -137,44 +117,35 @@ func (s *VMTServer) Run(_ []string) error {
 	// 	&clientcmd.ClientConfigLoadingRules{ExplicitPath: s.Kubeconfig},
 	// 	&clientcmd.ConfigOverrides{ClusterInfo: clientcmdapi.Cluster{Server: s.Master}}).ClientConfig()
 
-	kubeconfig, err := clientcmd.BuildConfigFromFlags(s.Master, s.Kubeconfig)
+	kubeConfig, err := clientcmd.BuildConfigFromFlags(s.Master, s.Kubeconfig)
 	if err != nil {
 		glog.Errorf("Error getting kubeconfig:  %s", err)
 		return err
 	}
 	// This specifies the number and the max number of query per second to the api server.
-	kubeconfig.QPS = 20.0
-	kubeconfig.Burst = 30
+	kubeConfig.QPS = 20.0
```

Comment:
**QPS** indicates the maximum QPS to the master from this client.
Maximum **burst** for throttle.
You can find more[here](https://github.com/kubernetes/client-go/blob/master/rest/config.go#L103)